utils/utils.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func GetOwnerRepoPull(ref, repo string) (string, string, int) {
+	// get pull request files
```

Comment:
Again instead of the comment we can use a better name in the var and get rid of the comment